dist/mac/Info.plist

```diff
@@ -37,7 +37,7 @@
 		</dict>
 	</array>
 	<key>CFBundleName</key>
-	<string>qBittorrent</string>
+	<string>@EXECUTABLE@</string>
```

Comment:
`CFBundleName` is how macOS determines the display name of the program ("application" in macOS), or other similar compiled directory.  `@EXECUTABLE@` is the macOS equivalent of a Windows "environment variable", in this case referring to the filename of the actual binary at the heart of the bundle.  In this case the file would be the one named qBittorrent (or something similar) at the core executable directory `Mac OS` (I may be getting the spacing and capitalization wrong.  I haven't played on a Mac environment for months.)  

Background:  
Programs on macOS are not single files but uncompressed directories with a tagged internal structure and a `.app` extension on the main directory.  Apple calls these "bundles" or "packages". `Info.plist` tells macOS how to interpret that directory as a bundle (i.e. how to launch the program).